### DISCUSSION TITLE
How to check similarity score in milvus image similarity provided code?

### DISCUSSION THREAD

**Discussion by bhuvneshsaini** [milvus-io/milvus#23364] [Q&A and General discussion]
How to check similarity score in milvus image similarity provided code?

**Comment by yhmo** [comment]
Milvus search() API returns a list of pair(id to distance), the distance is calculated by Euclidean distance or inner product, it is not a score.

**Reply by bhuvneshsaini** [reply]
can you give me the code link for reference as I have already tried but unable to get.

**Reply by yhmo** [reply]
In the [bootcamp reverse_image_search](https://github.com/milvus-io/bootcamp/blob/master/solutions/image/reverse_image_search), 

The app calls milvus API to do ANN search here:
https://github.com/milvus-io/bootcamp/blob/ac948034bb6e547875fcf56b312feabca3883e47/solutions/image/reverse_image_search/server/src/milvus_helpers.py#L90

The search API returns an array of id-distance pairs, the app gets the id/distance from the result and lookup image path from MySQL by the id, eventually it shows a list of most similar images on the frontend. Here it parses the result:
https://github.com/milvus-io/bootcamp/blob/ac948034bb6e547875fcf56b312feabca3883e47/solutions/image/reverse_image_search/server/src/operations/search.py#L9C14-L9C14
